### ISSUE TITLE
[Feature Request]: `{nest_exploratory_dev}`  `Add **anyna** variable` phrasing

### ISSUE THREAD

**Issue by m7pr** [insightsengineering/teal.gallery#139]
https://genentech.shinyapps.io/nest_exploratory_dev/ -> Missing Data tab

I would rephrase to either `Add "anyna" variable` since `**` does not support bolding. 
Or I would rephrase to `Add "Any NA" variable` to make it more understandable, without the need to hover over the tooltip.

<img width="185" alt="image" src="https://github.com/insightsengineering/teal.gallery/assets/133694481/0a1030cb-33dd-4ba3-8923-cf099d23c5c0">

**Comment by chlebowa** [comment]
I believe this issue belongs in `teal.modules.general` as it arises in `tm_missing_data`.

**Comment by m7pr** [comment]
moving to here https://github.com/insightsengineering/teal.modules.general/issues/637

**Comment by pawelru** [comment]
psstt I will tell you a little secret
You don't have to close and open -> there is a dedicated feature to transfer issues between repos. The advantage is that it will keep the discussion among other metadata - sometimes there are relevant posts that we want to keep.
![image](https://github.com/insightsengineering/teal.gallery/assets/12943682/e231950c-7016-4783-809c-95ccac013e53)

Please make use of it. I do expect that our users will be opening issues in `teal` whereas they will actually belong elsewhere. That's fine because we cannot expect them to master the dependency tree. In such case close&reopen will also turn off OP notifications about close or questions.

**Comment by m7pr** [comment]
Ah thanks. Never paid too much attention to the bottom part of the right sidebar panel in here. Will use that for sure
